codacy/requirements.yaml

```diff
@@ -26,85 +26,85 @@ dependencies:
   git: git@github.com:codacy/kube-fluentd-operator.git
 
 - name: codacy-ingress
-  version: ">=0.1.1"
+  version: "0.8.0"
   repository: https://charts.codacy.com/stable
   git: git@bitbucket.org:qamine/codacy-ingress.git
   condition: codacy-ingress.create
 
 - name: portal
-  version: ">=6.7.1"
+  version: "6.20.1"
   repository: https://charts.codacy.com/stable
   git: git@bitbucket.org:qamine/portal.git
   condition: portal.create
 
 - name: ragnaros
-  version: ">=15.2.2"
+  version: "23.20.3"
   repository: https://charts.codacy.com/stable
   git: git@bitbucket.org:qamine/ragnaros.git
   condition: ragnaros.create
 
 - name: activities
-  version: ">=1.3.0"
+  version: "1.7.2"
   repository: https://charts.codacy.com/stable
   git: git@bitbucket.org:qamine/codacy-activities.git
   condition: activities.create
 
 - name: remote-provider-service
-  version: ">=8.6.0"
+  version: "8.25.1"
   repository: https://charts.codacy.com/stable
   git: git@github.com:codacy/remote-provider-service.git
   condition: remote-provider-service.create
 
 - name: hotspots-api
-  version: ">=1.4.1"
+  version: "1.6.6"
   repository: https://charts.codacy.com/stable
   git: git@bitbucket.org:qamine/hotspots-api.git
   condition: hotspots-api.create
 
 - name: hotspots-worker
-  version: ">=1.3.0"
+  version: "1.6.4"
   repository: https://charts.codacy.com/stable
   git: git@bitbucket.org:qamine/hotspots-worker.git
   condition: hotspots-worker.create
 
 - name: listener
-  version: ">=7.11.0"
+  version: "7.22.0"
```

Comment:
Only version bumped since last release `2.1.0`. All the others were just fixed manually to make sure they would not pick newer ones.